platform/mellanox/issu-version/Makefile

```diff
@@ -19,15 +19,10 @@ SHELL = /bin/bash
 .SHELLFLAGS += -e
 
 MAIN_TARGET = issu-version
-BUSTER_FILES_PATH = /sonic/target/files/buster
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
-ifneq (,$(findstring buster,$(DEST)))
 	sx_sdk --version | egrep -o 'ISSU [0-9]+' > $(DEST)/$(MAIN_TARGET) || {
```

Comment:
Can you please fix the indentation?

Reply:
I believe the indentation here is correct. Under a given target we indent 4 spaces to show the code that is being executed for that target. 

When I added the ifneq I added it and didn't change the indentation.